docs/guide/train_autopilot.md

```diff
@@ -27,6 +27,16 @@ folder from the raspberry pi.
 rsync -r pi@<your_pi_ip_address>:~/mycar/data/  ~/mycar/data/
 ```
 
+## Data augmentation
+
+Data augmentation is an optional step, but can be beneficial to prevent overfitting. One of the straightforward approaches is to horizontally flop the image and set steering angle to the opposite value.
```

Comment:
Please add some external link for overfitting problem, like: https://en.wikipedia.org/wiki/Overfitting 
or 
https://www.youtube.com/watch?v=u73PU6Qwl1I